apps/dashboard/src/components/Plans/List.tsx

```diff
@@ -1,20 +1,43 @@
 import React, { useMemo, useState, useCallback } from "react";
-import { ActionsMenu, Loading, useFormatMessage } from "@litespace/luna";
-import { IPlan } from "@litespace/types";
+import {
+  ActionsMenu,
+  Loading,
+  toaster,
+  useFormatMessage,
+} from "@litespace/luna";
+import { IPlan, Void } from "@litespace/types";
 import { UseQueryResult } from "@tanstack/react-query";
 import { createColumnHelper } from "@tanstack/react-table";
 import Price from "@/components/Plans/Price";
 import Table from "@/components/Plans/Table";
 import BooleanField from "@/components/common/BooleanField";
 import DateField from "@/components/common/DateField";
 import PlanForm from "@/components/Plans/PlanForm";
+import { useDeletePlan } from "@litespace/headless/plans";
 
 const List: React.FC<{
   query: UseQueryResult<IPlan.MappedAttributes[], Error>;
-}> = ({ query }) => {
+  refresh: Void;
+}> = ({ query, refresh }) => {
   const intl = useFormatMessage();
   const [plan, setPlan] = useState<IPlan.MappedAttributes | null>(null);
   const close = useCallback(() => setPlan(null), []);
+
+  const onSuccess = useCallback(() => {
+    toaster.success({ title: intl("dashboard.plan.delete.success") });
+    refresh();
+  }, [intl, refresh]);
```

Comment:
Add empty line please. 

```suggestion
  }, [intl, refresh]);
  
```